frontend/store.js

```diff
@@ -27,7 +27,8 @@ const initialState = {
     
     chat: {
         status: null,
-        log: []
+        log: [],
+        userNames: null
```

Comment:
Probably better to make it *empty array* by default. Thus you won't need to check *userNames !== null* in your *Chat.jsx* changes, and *propTypes* will validate properly with your *PropTypes.array.isRequired*.